ios/FluentUI/ActivityIndicator/ActivityIndicatorTokenSet.swift

```diff
@@ -15,16 +15,16 @@ import UIKit
     case xLarge
 }
 
-/// Design token set for the `ActivityIndicator` control.
-public class ActivityIndicatorTokenSet: ControlTokenSet<ActivityIndicatorTokenSet.Tokens> {
-    public enum Tokens: TokenSetKey {
-        /// The default color of the Activity Indicator.
-        case defaultColor
+public enum ActivityIndicatorToken: Int, TokenSetKey {
```

Comment:
Do they still need to be `TokenSetKey`, or could we switch to just `Int` and try to expose the tokens to ObjC, even if the `ControlTokenSet` hasn't been exposed yet?

Reply:
I guess it might need to be `Int, CaseIterable`

Reply:
Oh, looks like the global tokens are objc and `Int, TokenSetKey`, so we might be able to just add objc without any other changes.

Reply:
Correct, we could add `@objc` any time we want. The big problem is that, at least based on my earlier testing, that *tanks* our binary size. In fact, this entire PR came out of some experimentation I was doing re: exposing control tokens to ObjC.